rc/extra/autorestore.kak

```diff
@@ -7,31 +7,39 @@ def autorestore-restore-buffer -docstring "Restore the backup for the current fi
         buffer_basename="${kak_buffile##*/}"
```

Comment:
@mawww How come `basename` wasn't used here?

Reply:
No need to spawn an entire process for something a POSIX expansion can handle!